docs/guide/development/develop-deviceshifu.md

```diff
@@ -91,9 +91,9 @@ type DeviceShifu struct {
 	opcuaClient       *opcua.Client
 }
 ```
-Make a method to create an instance of the struct like:
+Create an instance of the struct:
 ```go
-// New This function creates a new Device Shifu based on the configuration
+// This function creates a new Device Shifu based on the configuration
```

Comment:
We need the `New` here, which refers to the function name. It is go doc's convention. Besides, this part is copied from codebase, let's keep it consistent.